src/script/chem/molfile/molfile.js

```diff
@@ -386,6 +387,53 @@ Molfile.prototype.writeBondBlock3000 = function (molecule) {
 	this.writeCR('M  V30 END BOND');
 };
 
+/**
+ * @param {Struct} molecule
+ *
+ * Example collection block:
+ * 	M  V30 BEGIN COLLECTION
+ * 	M  V30 MDLV30/STEABS ATOMS=(2 3 4)
+ * 	M  V30 MDLV30/STEREL4 ATOMS=(1 2)
+ * 	M  V30 MDLV30/STERAC5 ATOMS=(2 1 6)
+ * 	M  V30 END COLLECTION
+ */
+Molfile.prototype.writeCollectionBlock3000 = function (molecule) {
+	const { abs, rac, rel } = molecule.enhancedStereo;
+	this.writeCR('M  V30 BEGIN COLLECTION');
+
+	// Absolute stereochemistry collection
+	// eg: M  V30 MDLV30/STEABS ATOMS=(2 3 4)
+	if (abs.length) {
+		const atomIndices = abs.map((id) => id + 1).join(' ');
+		let absoluteDetails = this.getMol3000Prefix();
```

Comment:
@apar03 , any naming conventions for the variable names? Not sure if `absolueDetails` and `absoluteDetailsLines` are proper.
Also in the future we probably need to redesign how we use the writer, like maybe add an util for your logic of wrapping the text every 70chars.

Reply:
> @apar03 , any naming conventions for the variable names? Not sure if absolueDetails and absoluteDetailsLines are proper.
- For now let's go with this.

> Also in the future we probably need to redesign how we use the writer, like maybe add an util for your logic of wrapping the text every 70chars.

- I agree with adding a util for the text wrapping, we could do that as part of a future ticket.